### PR TITLE
Rename proto name's for clarity and consistency

### PR DESCRIPTION
**Pull request by kalensk**
Rename the following:
idl.UpgradeSteps => idl.Substep
idl.StepStatus => idl.Status
idl.UpgradeStepStatus => idl.SubstepStatus

Test pipeline is running...

### CODE REVIEW

File: idl/cli_to_hub.proto [hunk 1]
```diff
@@ -35,14 +35,12 @@ message RestartAgentsReply {
...
-    StepStatus status = 2;
+message SubstepStatus {
+  Substep step = 1;
+  Status status = 2;
 }
 
-// TODO: rename to Substep
-enum UpgradeSteps {
+enum Substep {
     UNKNOWN_STEP = 0; // http://androiddevblog.com/protocol-buffers-pitfall-adding-enum-values/
```
Comment (jchampio) [inline idl/cli_to_hub.proto]: Can this be just `UNKNOWN`...

File: idl/cli_to_hub.proto [hunk 2]
```diff
@@ -58,8 +56,7 @@ enum UpgradeSteps {
     RECONFIGURE_PORTS = 12;
 }
 
-// TODO: rename to Status
-enum StepStatus {
+enum Status {
     UNKNOWN_STATUS = 0; // http://androiddevblog.com/protocol-buffers-pitfall-adding-enum-values/
```
Comment (jchampio) [inline idl/cli_to_hub.proto]: ...along with this?

File: idl/cli_to_hub.proto (hunk 1 above)
Comment (kalensk) [reply idl/cli_to_hub.proto]: I wish...doing so resulted in the following  :(
```
cli_to_hub.proto:60:3: "UNKNOWN" is already defined in "idl".
cli_to_hub.proto:60:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "UNKNOWN" must be unique within "idl", not just within "Status".
```

Comment (jchampio) [reply idl/cli_to_hub.proto]: _twitch_

Thanks for trying.

**Pull request merged**